tests/Microsoft.Identity.Test.Common/TestCategories.cs

```diff
@@ -13,5 +13,7 @@ public static class TestCategories
         public const string Selenium = "Selenium";
         public const string LabAccess = "LabAccess";
 
+        public const string ADFS = "ADFS";
```

Comment:
How about adding "cloud"

Reply:
Do you mean add a separate constant here "Public Cloud" ?

Reply:
yeah, there are other tests that are public cloud tests in the file

Reply:
actually, maybe not. ADFS should be fine